onnx/defs/rnn/defs.cc

```diff
@@ -71,6 +90,15 @@ allows to make spec more uniform.
     .Attr("num_layers", "Numbers of RNN layers in the stack, default 1",
           AttrType::INT)
     .Attr("hidden_size", "Number of neurons in the hidden layer", AttrType::INT)
+    .Attr("activations", "A list of activation functions for the gates. Typical activation "
```

Comment:
you need to describe the order of activation functions (probably the same as the gates order in weights blob below). And say that default version just assumes stuff from the formula above.

Reply:
Done.